### PR TITLE
Add rollup

### PR DESCRIPTION
**Pull request by taras**
Added rollup to generate dist files

### CODE REVIEW

File: src/union.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import { create, valueOf } from 'microstates';
-import expect from 'expect';
+import * as M from 'microstates';
+
+const { create, valueOf } = M;
```
Comment (cowboyd) [inline src/union.js:3]: I kinda hesitate to ask, but why is this necessary?

Comment (taras) [reply src/union.js:3]: When I try to use `import { create, valueOf } from 'microstates'`, I get an error says that `valueOf` is not a module. I have suspicion that it might be treating `valueOf` module as `{}.valueOf`. I'm not 100%, but all other exports seemed fine.

Comment (cowboyd) [reply src/union.js:3]: Hmmm... Is this going to be a problem for libraries that use microstates generally?

Comment (taras) [reply src/union.js:3]: This seems to only be a problem when using Microstates as a dependency with Rollup. I think it's something to do with rollup trying to use CJS.
